ember-app/app/models/new/board.js

```diff
@@ -92,6 +92,16 @@ var Board = Model.extend({
       return combined;
     }
   }),
+  issuesById: function(){
+    return _.groupBy(this.get('issues'), (issue)=> {
+      return issue.get('id');
+    });
+  }.property('issues.[]'),
```

Comment:
To clarify my Ember understanding, this means the the `issuesById` function will be re-run any time the `issues` collection changes (i.e. a message comes in) but otherwise the value will be reused?